src/test/java/org/opensearch/searchrelevance/action/judgment/JudgmentsIT.java

```diff
@@ -120,4 +178,46 @@ public void testMainActions_whenImportReadJudgments_thenSuccessful() {
             )
         );
     }
+
+    @SneakyThrows
+    public void testListJudgments_filteringByStatus_thenSuccessful() {
+        String createBase = """
+            {
+              "name": "FilterTest",
+              "type": "IMPORT_JUDGMENT",
+              "judgmentRatings": [
+                {
+                  "query": "shoes",
+                  "ratings": [
+                    { "docId": "abc", "rating": "1.0" }
+                  ]
+                }
+              ]
+            }
+            """;
+
+        // Create 4 judgments
+        String completedId = createJudgment(createBase);
+        String processingId = createJudgment(createBase);
+        String errorId = createJudgment(createBase);
+        String timeoutId = createJudgment(createBase);
+
+        Thread.sleep(DEFAULT_INTERVAL_MS);
```

Comment:
Thread.sleep always makes me nervous, but if this is a pattern we use, then fine!